### PR TITLE
common: adjust advice on what do do in case of internal errors

### PR DESCRIPTION
**Pull request by peterbarker**
We do NOT want people to treat these as if they were trivial problems.

### CODE REVIEW

**Comment by Hwurzburg** [comment]
spelling....development

**Comment by rmackay9** [comment]
Let's add some additional advice:

- add link to the issues list. https://github.com/ArduPilot/ardupilot/issues
- include the exactly internal error number

**Comment by rmackay9** [comment]
We decided to create too sticky issues on the issues list.  one for watchdogs and one for internal errors.  We will direct users to comment on these issues instead of creating new issues.

**Comment by peterbarker** [comment]
@rmackay9 So will I close this?

File: common/source/docs/common-prearm-safety-checks.rst [hunk 1]
```diff
@@ -210,7 +210,7 @@ System:
 
 **Param storage failed**: A check of reading the parameter storage area failed.
 
-**Internal errors (0xx)**: An internal error has occurred. Try rebooting.
+**Internal errors (0xx)**: An internal error has occurred. Report to ArduPilot development team.
```
Comment (Hwurzburg) [inline common/source/docs/common-prearm-safety-checks.rst:213]:
```suggestion
**Internal errors (0xx)**: An internal error has occurred. Report to ArduPilot development team `here <https://github.com/ArduPilot/ardupilot/issues/15916>`_
```

**Comment by Hwurzburg** [comment]
merging...thanks

**Pull request merged**